SiteAuditCommands.php

```diff
@@ -75,6 +75,23 @@ public function auditReports(
         $this->init();
 
         $checks = $this->interimInstantiateChecks($this->createRegistry($options));
+
+        $settings_excludes = \Drupal::config('site_audit')->get('opt_out');
+
+        if(!empty($settings_excludes)) {
+          $settings_excludes = array_keys($settings_excludes);
+
+          if (is_array($options['skip'])) {
+            $options['skip'] += $settings_excludes;
```

Comment:
Pull the `explode` of $options['skip'] out of filterSkippedChecks and put it at the top of this method so that you can simplify this logic.

Reply:
Resolved